### ISSUE TITLE
Style Header Component 

### ISSUE THREAD

**Issue by ezioda004** [UdacityFrontEndScholarship/quiz-app#32]
## Style the Header Component as shown in this picture.
![image](https://user-images.githubusercontent.com/20020273/38775935-f9971c9a-40ab-11e8-9399-549a88bc98d2.png)

## For the logo use this image. 
![image](https://user-images.githubusercontent.com/20020273/38776047-403a93d2-40ae-11e8-8f8e-7797acc11e4b.png)

How to get started? Read our [project setup instructions](https://github.com/UdacityFrontEndScholarship/quiz-app/blob/master/CONTRIBUTING.md#project-setup-instructions).
Fork the project, navigate to directory quiz-app => src => Component => Header.css and Header.js

### All you need to do is write normal CSS inside Header.css.
We are using Bootstrap-4 for this project but feel free to use native CSS wherever required.

Note:

1. If you use any self closing tags such as `<img>`, `<input>`, `<br>`
2. Make sure you always close them like this `<img />`, `<input />`, `<br />`

If you add a class to an element example
```
<div  class = "nav navbar"> 
    <!-- rest of your code -->
</div>
```
Make sure you use className instead of class. Example
```
<div  className = "nav navbar"> 
    <!-- rest of your code -->
</div>
````

This is fairly easy and straightforward to do, please comment on this issue and I'll assign it to you.

**Comment by kadumuri1994** [comment]
Can I try this?

**Comment by ezioda004** [comment]
@muditasisodia Absolutely! Assigned to you.
If you get stuck anywhere, feel free to ping us here or on slack. We'll assist you the best way possible.

**Comment by muditasisodia** [comment]
Great, thanks!

**Comment by muditasisodia** [comment]
Should the navbar be light grey as it is of class bg-light in the source code or should I set it to white or transparent as it is in the above image?

**Comment by ezioda004** [comment]
@muditasisodia You can definitely change the source code so it looks like the image.
The provided code is just skeleton, if required you can make any changes.

**Comment by lalitmee** [comment]
Hey @muditasisodia, I think you can keep it transparent as it is in the above image. Don't worry about the code, the code is written by the people only to put the elements there. I would say that you can try to make as it is appearing in the image. :+1:

**Comment by muditasisodia** [comment]
Okay thanks

**Comment by ezioda004** [comment]
Resolved #37 
Closing issue.
